roboconf-dist-rpm-dm/src/files/bin/init.d

```diff
@@ -0,0 +1,148 @@
+#!/bin/sh
+#
+#  Copyright 2014 Linagora, Université Joseph Fourier
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+### BEGIN INIT INFO
+# Provides:          roboconf-agent
+# Required-Start:    $remote_fs $syslog
+# Required-Stop:     $remote_fs $syslog
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: The startup script for Roboconf agent
+# Description:       This file should be used to construct scripts to be
+#                    placed in /etc/init.d.
+### END INIT INFO
+
+#
+# Author: Vincent ZURCZAK <vincent.zurczak@linagora.com>
+#
+
+# Do NOT "set -e"
+
+# PATH should only include /usr/* if it runs after the mountnfs.sh script
+PATH=/sbin:/usr/sbin:/bin:/usr/bin
+DESC="Roboconf Agent"
+NAME=roboconf-agent
+DAEMON=/opt/roboconf-agent/bin/karaf
+SCRIPTNAME=/etc/init.d/$NAME
+RUNDIR=/var/run/$NAME
+PIDFILE=/var/run/$NAME.pid
+export KARAF_HOME=/opt/roboconf-agent
```

Comment:
Does the agent start with the system?
When you install the agent with such a package, restart your VM then and verify the agent is running.
On Debian systems, we use `/etc/init.d/roboconf-agent status` to verify it.